teams/wg-parallel-rustc.toml

```diff
@@ -19,6 +19,9 @@ alumni = [
     "spastorino",
 ]
 
+[permissions]
+perf = true
+
```

Comment:
Unless the `perf` permission also implies the right to trigger `try` builds (I don't believe it's the case), I think the following permission is also going to be needed.

```toml
bors.rust.try = true
```

Reply:
Thanks for pointing out! I have added the line